SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Common/LinuxScripts.cs

```diff
@@ -16,7 +16,7 @@ shift 2
 
 # 2>/dev/null silences errors by redirecting stderr to the null device. This is done to prevent errors when a machine attempts to add the same user more than once.
 mkdir -p /improbable/logs/UnrealWorker/
-useradd $NEW_USER -m -d /improbable/logs/UnrealWorker/Logs 2>/dev/null
+useradd $NEW_USER -m -d /improbable/logs/UnrealWorker 2>/dev/null
```

Comment:
Still think we should get rid of `2>/dev/null` or pipe it to a file. Could we add a ticket?